Source/Core/Core/PowerPC/Jit64IL/JitIL.cpp

```diff
@@ -310,9 +310,9 @@ void JitIL::FallBackToInterpreter(UGeckoInstruction _inst)
 
 void JitIL::HLEFunction(UGeckoInstruction _inst)
 {
+  ABI_PushRegistersAndAdjustStack({}, 0);
   ABI_CallFunctionCC(HLE::Execute, js.compilerPC, _inst.hex);
-  MOV(32, R(RSCRATCH), PPCSTATE(npc));
-  WriteExitDestInOpArg(R(RSCRATCH));
+  ABI_PushRegistersAndAdjustStack({}, 0);
```

Comment:
I'm not really acquainted with JIT code, but shouldn't one of these pop the registers.